submissions/vovan-zt/DOM/script.js

```diff
@@ -0,0 +1,97 @@
+        },
+        "node": 
+        {
+            "img": "icons/skills/node.js.svg",
+            "title": "Node",
+            "descr": " Node или Node.js — программная платформа, основанная на движке V8 (компилирующем JavaScript в машинный код), превращающая JavaScript из узкоспециализированного языка в язык общего назначения. Node.js добавляет возможность JavaScript взаимодействовать с устройствами ввода-вывода через свой API, написанный на C++, подключать другие внешние библиотеки, написанные на разных языках, обеспечивая вызовы к ним из JavaScript-кода. Node.js применяется преимущественно на сервере, выполняя роль веб-сервера, но есть возможность разрабатывать на Node.js и десктопные оконные приложения (при помощи NW.js, AppJS или Electron для Linux, Windows и macOS) и даже программировать микроконтроллеры (например, tessel, low.js и espruino). В основе Node.js лежит событийно-ориентированное и асинхронное (или реактивное) программирование с неблокирующим вводом/выводом.",
+        },
+        "jquery": 
+        {
+            "img": "icons/skills/jquery.svg",
+            "title": "Jquery",
+            "descr": " jQuery  — набор функций JavaScript, фокусирующийся на взаимодействии JavaScript и HTML. Библиотека jQuery помогает легко получать доступ к любому элементу DOM, обращаться к атрибутам и содержимому элементов DOM, манипулировать ими. Также библиотека jQuery предоставляет удобный API для работы с AJAX. Разработка jQuery ведётся командой добровольцев на пожертвования.",
+        },
+        
+    }
+
+    const menuList = document.querySelector('.menu__list'),
+        menuLink = document.querySelectorAll('.menu__link'),
+        mainContent = document.querySelector('.main__content');
+
+    function menuToggleClass (target) {
+        menuLink.forEach((link, i) => {
+            if (target == link) {
+                link.classList.add('menu__link-active');
+            } else {
+                link.classList.remove('menu__link-active');
+            }
+        });
+    }
+
+    function toggleContent (target) {
+        for(let key in dataContent) {
+            if (target.getAttribute('data-content') == key) {
+                mainContent.firstChild.remove();
+                const element = document.createElement('div');
+                element.classList.add('main__item');
+                element.classList.add('box');
+
+                element.innerHTML = `
+                <div class="main__item-icon">
+                    <img src="${dataContent[key].img}" alt="${key}">
+                </div>
+                <h2 class="main__item-title">
+                    ${dataContent[key].title}
+                </h2>
+                <div class="main__item-descr">
+                    ${dataContent[key].descr}
+                </div>
+                `;
+                mainContent.append(element);
+            }    
+        }
+    }
+
+    menuList.addEventListener('click', ({target}) => {
```

Comment:
Separate listener and function.
`menuList.addEventListener('click', showContent)`